playground/GarnetJSON/JsonObject.cs

```diff
@@ -125,54 +166,83 @@ public bool TrySet(string path, string value, ILogger? logger = null)
 
         private void Set(string path, string value)
         {
-            var tokens = jObject.SelectTokens(path);
-            if (!tokens.Any())
+            // Find all items matching JSON path
+            var jPath = JsonPath.Parse(path);
```

Comment:
Should https://docs.json-everything.net/api/JsonPath.Net/PathParseException/ be caught as well for parsing errors?